ui/app/components/app/modals/account-details-modal.js

```diff
@@ -86,7 +90,16 @@ AccountDetailsModal.prototype.render = function () {
       h(Button, {
         type: 'secondary',
         className: 'account-modal__button',
-        onClick: () => global.platform.openWindow({ url: genAccountLink(address, network) }),
+        onClick: () => {
+          let accountLink
+          if (blockExplorerUrl) {
+            accountLink = `${blockExplorerUrl}/address/${address}`
```

Comment:
i would turn this into a `genTxLink` util that checks for `rpcPrefs` and default to etherscan, that way you can re-use it in `transaction-list-item-details.component.js:41`

Reply:
Done in 359cb7506126d3d0bcf6ec55af9fc3753898ca7a